neqo-transport/src/connection.rs

```diff
@@ -551,6 +560,11 @@ impl Connection {
         }
 
         self.tps.borrow_mut().remote_0rtt = Some(tp);
+
+        if smoothed_rtt > GRANULARITY.as_millis() as u64 {
```

Comment:
```suggestion
        if Duration::from_millis(smoothed_rtt) > GRANULARITY {
```
Or you can take the above suggestion.